### PR TITLE
fix zh_hans and zh_hant-hk

### PR DESCRIPTION
**Pull request by helight**
compare the translations between zh_hans and zh_hant-hk.
1. fix some words
2. translate some words that not translated

### CODE REVIEW

**Comment by helight** [comment]
@chenrui333 pls take a look about this pr.

**Comment by maciaszczykm** [comment]
@hwdef @tanjunchen @chenrui333 @zehuaiWANG Can one of you review and approve?

**Comment by chenrui333** [comment]
Sure, taking a look now.

**Review by chenrui333 — APPROVED** [review]
lgtm, thanks!

File: i18n/zh-Hans/messages.zh-Hans.xlf [hunk 1]
```diff
@@ -2030,39 +2030,39 @@
       </trans-unit>
       <trans-unit id="90ae2a4959c979abc049f4652f59b85e993e16cd" datatype="html">
         <source>CPU requests (cores)</source>
-        <target>CPU 最低需求 (cores)</target>
+        <target>CPU 下限 (cores)</target>
```
Comment (chenrui333) [inline i18n/zh-Hans/messages.zh-Hans.xlf:2033]: 我之前的理解 一直都是 需求。不过 下限 也 make sense。

Comment (helight) [reply i18n/zh-Hans/messages.zh-Hans.xlf:2033]: 嗯，这里是和香港这个版本做了一个统一

File: i18n/zh-Hans/messages.zh-Hans.xlf [hunk 2]
```diff
@@ -3091,7 +3091,7 @@
                 Please select the kubeconfig file that you have created to configure access to the cluster. To find out more about how to configure and use kubeconfig file, please refer to the <x id="START_LINK" ctype="x-a" equiv-text="&lt;a>"/>Configure Access to Multiple Clusters<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/> section.
               </source>
         <target>
-                请选择您创建的kubeconfig文件以配置对集群的访问权限。  要了解有关如何配置和使用 kubeconfig 文件的更多信息， 请参阅<x id="START_LINK" ctype="x-a" equiv-text="&lt;a>"/>Configure Access to Multiple Clusters<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/> 部分.
+                请选择您创建的 kubeconfig 文件以配置对集群的访问权限。  要了解有关如何配置和使用 kubeconfig 文件的更多信息， 请参阅<x id="START_LINK" ctype="x-a" equiv-text="&lt;a>"/>Configure Access to Multiple Clusters<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/> 部分.
```
Comment (tanjunchen) [inline i18n/zh-Hans/messages.zh-Hans.xlf:3094]: 您 -> 你   就行，不要那么正式 保持上下文一致

Comment (tanjunchen) [reply i18n/zh-Hans/messages.zh-Hans.xlf:3094]: /lgtm
Thanks

**Pull request merged**